docker/docker-compose-ssl.yml

```diff
@@ -0,0 +1,38 @@
+version: '3'
+services:
+  saltmaster-local-ssl:
+    image: erwindon/saltgui-saltmaster-ssl:3007.4
+    hostname: saltmaster-local-ssl
```

Comment:
same applies here.
it is of course possible to only change this one, but so far the container names and the hostnames are equal, let's keep it that way.